dashbuilder-client/dashbuilder-cms-client/src/main/java/org/dashbuilder/client/cms/screen/explorer/ContentExplorerScreen.java

```diff
@@ -0,0 +1,142 @@
+
+@ApplicationScoped
+@WorkbenchScreen(identifier = ContentExplorerScreen.SCREEN_ID)
+public class ContentExplorerScreen {
+
+    public static final String SCREEN_ID = "ContentExplorerScreen";
+
+    public interface View extends UberView<ContentExplorerScreen> {
+
+        void showPerspectives(IsWidget perspectivesExplorer);
+
+        void showMenus(IsWidget menusExplorer);
+
+        void setPerspectivesName(String name);
+
+        void setMenusName(String name);
+
+        void setCreateName(String name);
+
+        void setCreateMenuVisible(boolean visible);
+
+        void addCreateMenuEntry(String name, Command onClick);
+    }
+
+    View view;
+    NavigationManager navigationManager;
+    PerspectivesExplorer perspectiveExplorer;
+    NavTreeEditor navTreeEditor;
+    NewPluginPopUp newPluginPopUp;
+    PluginController pluginController;
+    ContentManagerI18n i18n;
+    Event<NotificationEvent> workbenchNotification;
+
+    public ContentExplorerScreen() {
+    }
+
+    @Inject
+    public ContentExplorerScreen(View view,
+                                 NavigationManager navigationManager,
+                                 PerspectivesExplorer perspectiveExplorer,
+                                 NavTreeEditor navTreeEditor,
+                                 NewPluginPopUp newPluginPopUp,
+                                 PluginController pluginController,
+                                 ContentManagerI18n i18n,
+                                 Event<NotificationEvent> workbenchNotification) {
+        this.view = view;
+        this.navigationManager = navigationManager;
+        this.perspectiveExplorer = perspectiveExplorer;
+        this.navTreeEditor = navTreeEditor;
+        this.newPluginPopUp = newPluginPopUp;
+        this.pluginController = pluginController;
+        this.i18n = i18n;
+        this.workbenchNotification = workbenchNotification;
+        this.view.init(this);
+    }
+
+    @AfterInitialization
+    private void init() {
+        navTreeEditor.setLiteralPerspective(i18n.capitalizeFirst(i18n.getPerspectiveResourceName()));
+        view.setPerspectivesName(i18n.capitalizeFirst(i18n.getPerspectivesResourceName()));
+        view.setMenusName(i18n.getContentExplorerMenus());
+        view.setCreateName(i18n.getContentExplorerNew());
+        view.setCreateMenuVisible(pluginController.canCreatePerspectives());
+        view.addCreateMenuEntry(i18n.capitalizeFirst(i18n.getPerspectiveResourceName()), this::createNewPerspective);
+        gotoPerspectives();
+    }
+
+    @WorkbenchPartTitle
+    public String getTitle() {
+        return i18n.getContentExplorer();
+    }
+
+    @WorkbenchPartView
+    public IsWidget getWidget() {
+        return view;
+    }
+
+    public void createNewPerspective() {
+        // TODO: customize new perpsective popup title
+        // newPluginPopUp.show(PluginType.PERSPECTIVE_LAYOUT, i18n.getContentManagerHomeNewPerspectiveButton());
```

Comment:
It was kept as a todo since it requires a change in UF.  I wouldn't worry too much about this for the time being as it is just a tweak.